### PR TITLE
Update action to run on node20

### LINKED ISSUE #121: Upgrade action to node 20
We're seeing a warning in our builds with this action:

> Node.js 16 actions are deprecated. Please update the following actions to use Node.js 20: dblock/create-a-github-issue@v3. For more information see: https://github.blog/changelog/2023-09-22-github-actions-transitioning-from-node-16-to-node-20/.

Is it possible to upgrade this action to target Node 20?

### PR DESCRIPTION
**Pull request by pkwarren**
Fixes #121.

### CODE REVIEW

File: tsconfig.json [hunk 1]
```diff
@@ -1,6 +1,3 @@
 {
-  "extends": "@tsconfig/node16/tsconfig.json",
-  "compilerOptions": {
-    "esModuleInterop": true
```
Comment (pkwarren) [inline tsconfig.json:4]: This is set to true in the base config: https://github.com/tsconfig/bases/blob/be6b3bb160889347b8614e8d18e1e88c40f8ecc9/bases/node20.json#L12

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
 ### 3.1.2 (Next)
 
-* Your contribution here.
```
Comment (dblock) [inline CHANGELOG.md:3]: Put back that line pls.

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,6 +1,6 @@
 ### 3.1.2 (Next)
```
Comment (dblock) [inline CHANGELOG.md:1]: Lets bump the version to 3.2.0 since we have a new "feature" as part of this PR.

**Review by dblock — APPROVED** [review]
Let's bump the version.

**Review by dblock — CHANGES_REQUESTED** [review]
We also should bump it in CI, shouldn't we?

https://github.com/dblock/create-a-github-issue/blob/main/.github/workflows/ci.yml#L15

**Review by dblock — CHANGES_REQUESTED** [review]
The version increment in package.json/lock (https://github.com/dblock/create-a-github-issue/blob/68381210a25b5c64fe67add12d8f4713230ce53b/package.json#L3), too, please?

**Comment by dblock** [comment]
Thanks for contributing!

**Comment by dblock** [comment]
@pkwarren If you're looking for more work, finishing https://github.com/dblock/create-a-github-issue/issues/110 would be grand!

**Pull request merged**